src/experimental/shader-modules/index.js

```diff
@@ -1,16 +1,15 @@
-import {registerShaderModules} from '../shader-tools';
+// import * as fp64 from './fp64';
 
-import * as fp64 from './fp64';
-
-import * as fog from './fog';
-import * as picking from './picking';
+// import * as fog from './fog';
+// import * as picking from './picking';
 // import * as lighting from './lighting';
-import * as material from './material';
+// import * as material from './material';
 
-registerShaderModules([
-  fp64,
-  fog,
-  picking,
-  // lighting,
-  material
-]);
+// TODO - This probably defeats tree-shaking
```

Comment:
I know how tree shaking works, but I feel like giving this responsibility to the user is a bit of overhead we should be able remove.

By simply registering each module one by one in their respective file, we would prevent that. Maybe just rename `registerShaderModules` into `registerShaderModule` and only pass only one module instead of a list.

Or is there something I'm missing?